src/stacktraces.cpp

```diff
@@ -536,12 +536,9 @@ static std::map<void*, std::shared_ptr<std::vector<uint64_t>>> g_stacktraces;
 
 #if CRASH_HOOKS_WRAPPED_CXX_ABI
 // These come in through -Wl,-wrap
-// It only works on GCC
 extern "C" void* __real___cxa_allocate_exception(size_t thrown_size);
 extern "C" void __real___cxa_free_exception(void * thrown_exception);
-#if __clang__
-#error not supported on WIN32 (no dlsym support)
-#elif WIN32
+#if WIN32
```

Comment:
also, feels like these changes should be in another PR? This PR should just be adding clang-15 and thread-safety

Reply:
The fixes for stacktraces are directly aimed to resolve problems with a Clang/LLVM target, which didn't exist in our CI suite until this PR, so it was retained here. The changes weren't significant enough to warrant its own PR and aren't verifiable without this PR's merger (sans stracktrace changes)